contracts/liquidityMining/KyberFairLaunch.sol

```diff
@@ -196,69 +185,53 @@ contract KyberFairLaunch is IKyberFairLaunch, PermissionAdmin, ReentrancyGuard {
       'renew: invalid pool state to renew'
     );
 
-    // checking data of new pool
-    require(
-      _startBlock > block.number && _endBlock > _startBlock, 'add: invalid blocks'
-    );
+    // check if pool has not started or already ended
```

Comment:
should be `// check new pool params` or keep the previous comment `// checking data of new pool`